pkg/qe-tests/cypress/integration/models/mapping.ts

```diff
@@ -98,7 +111,6 @@ export class Mapping {
     this.selectSourceProvider(sProviderName);
     this.selectTargetProvider(tProviderName);
     this.createMappingPeer(mappingPeer);
-    // cy.wait(2000);
```

Comment:
Remove unused line.

Reply:
Strange, I don't have it in my code.
Listing of this method is:
```
  protected createDialog(mappingData: MappingData): void {
    const { name, mappingPeer, sProviderName, tProviderName } = mappingData;
    cy.wait(2 * SEC);
    clickByText(button, createMapping);
    this.inputName(name);
    this.selectSourceProvider(sProviderName);
    this.selectTargetProvider(tProviderName);
    this.createMappingPeer(mappingPeer);
    clickByText(buttonModal, create);
    this.validateReady(mappingData);
  }
  ```

Reply:
You're right ! Sorry this line was removed. Never mind.